entrypoint.sh

```diff
@@ -845,8 +845,8 @@ chown ${GITLAB_USER}:${GITLAB_USER} ${GITLAB_BACKUP_DIR}
 
 # create the uploads directory
 mkdir -p ${GITLAB_DATA_DIR}/uploads/
-chmod 0750 ${GITLAB_DATA_DIR}/uploads/
-chown ${GITLAB_USER}:${GITLAB_USER} ${GITLAB_DATA_DIR}/uploads/
+chmod -R u=rwX,g=rX,o= ${GITLAB_DATA_DIR}/uploads/
+chown -R ${GITLAB_USER}:${GITLAB_USER} ${GITLAB_DATA_DIR}/uploads/
```

Comment:
Here we are only setting permissions on  the topmost `uploads/` directory so that this step does not effect the start up time of the container. If user notices issues, he/she is expected to run `app:sanitize` to fix them